### DISCUSSION TITLE
Welcome to EnTT discussions!

### DISCUSSION THREAD

**Discussion by skypjack** [skypjack/entt#608] [General]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

* Engage with other community members.
* Ask questions you’re wondering about.
* Share ideas, suggestions, requests for improvements.
* Submit your project _made with `EnTT`_ to share it, get feedback, or just to see it in the [showcases](https://github.com/skypjack/entt/wiki/EnTT-in-Action) list.
* Welcome others and are open-minded. Remember that this is a community we build together 💪.

And if all this is not enough, we are waiting for you on the [discord server](https://discord.gg/sDhsGnqG) or on the [gitter channel](https://gitter.im/skypjack/entt).
To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LiliumAtratum** [comment]
Thank you for settings this up! (and thank you for `entt`, obviously :) )
I will definitely keep an eye on this discussion as I am incorporating `entt` more and more in my code.
I find it much easier to reason about the logic of my program when it is expressed the the form of systems, some of which are pretty complex algorithms, with clearly defined dependencies (computed by previous systems) and otherwise nicely decoupled from the rest of the world.

**Reply by skypjack** [reply]
Thank you for using `EnTT` @LiliumAtratum and welcome here! 🙂

**Comment by MrJakobLaich** [comment]
does the discord server no longer exists?
(am on mobile right now and it just puts me into discord but doesn't open the usual "join this community" pop up)

**Reply by skypjack** [reply]
It exists, yeah, [here](https://discord.gg/cDpzEAN9) we go.

**Comment by joviol** [comment]
Hello,

I would like to use the dispatcher in my new project but one of my need is to be able to dispatch events from different threads (tasks).
I was wondering if the dispatching / queuing is thread safe or if I should add some global synchronization aground calls to the dispatcher?

**Reply by skypjack** [reply]
Please, create an issue or (even better) a new discussion for this kind of questions. Thanks.
